hack/dockerized

```diff
@@ -12,8 +12,8 @@ fi
 
 fail_if_cri_bin_missing
 
-kubevirt_builder_version="2510171022-dc7a64f2a5"
-default_kubevirt_builder_image="quay.io/kubevirt/builder:${kubevirt_builder_version}"
+kubevirt_builder_version="2510281240-82d14f4b41"
+default_kubevirt_builder_image="quay.io/kubevirt/builder:${kubevirt_builder_version}"	
```

Comment:
Apologies. Can you drop the whitespaces at the end?

Reply:
Sure, I just dropped it.